compose/ui/ui/src/webTest/kotlin/androidx/compose/ui/window/ComposeWindowLifecycleTest.kt

```diff
@@ -55,28 +58,27 @@ class ComposeWindowLifecycleTest {
             state = DefaultWindowState(document.documentElement!!)
         )
 
-        assertEquals(Lifecycle.State.RESUMED, lifecycleOwner.lifecycle.currentState)
+        val eventsChannel = Channel<Lifecycle.Event>(10)
+
+        lifecycleOwner.lifecycle.addObserver(object : LifecycleEventObserver {
```

Comment:
I was about to suggest the same, and in fact, there is already `ChannelLifecycleObserver` in desktop target.
I can refactor them both to share the logic later.